### DISCUSSION TITLE
Anyway to convert a multi channel EXR to a layered PSD in 32bit

### DISCUSSION THREAD

**Discussion by pixelvapour** [ImageMagick/ImageMagick#6541] [Help]
### ImageMagick version

7.1.1-6 

### Operating system, version and so on

Marc Ventura 13.4.1 Magick Q16-HDRI x86_64 21020

### Description

[Cone_tip.exr.zip](https://github.com/ImageMagick/ImageMagick/files/12251487/Cone_tip.exr.zip)

Is there anyway to take this 32bit multi channel EXR into a PSD format converting the multi channels into multiple PS layers still in 32 bit.
ProEXR used to do this fine but no longer works on Apple silicon.

**Comment by snibgo** [comment]
IM currently doesn't read or write extra channels in EXR files. It is limited to RGB and one alpha channel. See the source code https://github.com/ImageMagick/ImageMagick/blob/main/coders/exr.c
